src/client/AM.PA.MonitoringTool/SlackTracker/Data/SlackConnector.cs

```diff
@@ -0,0 +1,372 @@
+﻿// Created by Rohit Kaushik (f20150115@goa.bits-pilani.ac.in) at the University of Zurich
+// Created: 2018-07-09
+// 
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Text;
+using Shared;
+using Shared.Data;
+using Shared.Helpers;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using SlackTracker.Data.SlackModel;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json.Linq;
```

Comment:
just wondering, what is the difference of System.Linq to Newtonsoft.Json.Linq?

Reply:
Newtonsoft.Json.Linq just extends the System.Linq method for Json Objects (JToken, JArray)